integreat_cms/cms/templates/_base.html

```diff
@@ -211,12 +196,19 @@
                         {% translate "Imprint" %}
                     </a>
                 {% endif %}
-                {% if perms.cms.view_user or perms.cms.view_languagetreenode %}
+                {% if perms.cms.manage_translations or perms.cms.view_user or perms.cms.view_languagetreenode %}
                     <!-- Settings Section-->
                     <div class="min-h-3 pl-2">
                         <p class="font-bold">{% translate "Settings" %}</p>
                     </div>
                 {% endif %}
+                {% if perms.cms.manage_translations %}
+                    <a href="{% url 'translations_management' region_slug=request.region.slug %}"
+                       class="{% if current_menu_item == 'translations_management' %} active{% endif %}">
+                        <i icon-name='bot'></i>
+                        {% translate "Machine Translations" %}
```

Comment:
I agree with @timoludwig in the issue report, that we should widen the navigation bar, so "machine translation" fits in one row

Reply:
Oh no, I missed this point too 😓 Now it's widened.